src/main/java/frc/robot/RobotContainer.java

```diff
@@ -127,7 +129,7 @@ private void configureButtonBindings() {
         .withTimeout(1),
       new InstantCommand(() -> m_speakerTop.stop(), m_speakerTop),
       new RunCommand(() -> m_robotDrive.drive(AutoConstants.kAutoSpeed, 0, 0, false,false), m_robotDrive)
-        .until(() -> m_robotDrive.getPose().getX()>1)
+        .until(() -> m_robotDrive.getPose().getX()<-80*ConversionFactors.kInchesToMeters)
```

Comment:
You'll find it easier to use the Units package:
```suggestion
        .until(() -> m_robotDrive.getPose().getX()<-Units.inchesToMeters(80))
```